include/micm/configure/solver_config.hpp

```diff
@@ -39,7 +39,7 @@ namespace micm
       {
       }
 
-      virtual const char* what()
+      virtual const char* what() const noexcept override
       {
         return msg_;
```

Comment:
I think `override` specifier is for a function that overrides the virtual version of it, so I think we can remove it

Reply:
done!